cypress/support/commands.js

```diff
@@ -0,0 +1,32 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('setCookieSession', (value) => {
+    cy.session(value ,() => {
+        cy.visit(Cypress.env('baseUrl'))
+        cy.setCookie('cookieConsent', value)
+    })
+})
```

Comment:
```suggestion
})

```

Recomenda-se deixar uma linha em branco ao final de cada arquivo, para evitar que ferramentas como o GitHub exibam o ícone de `No newline at end of file`.

Isso é uma boa prática e convenção a ser seguida.